<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsDataAccessConfigurator</class>
 <widget class="QDialog" name="CwmsDataAccessConfigurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Data Access</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Datasource Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pqleDatasourceName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Datasource Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_pqcbDatasourceType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_pqWidgetStack">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOk">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wmsgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqcbDatasourceType</sender>
   <signal>currentIndexChanged</signal>
   <receiver>CwmsDataAccessConfigurator</receiver>
   <slot>DatasourceTypeChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOk</sender>
   <signal>clicked()</signal>
   <receiver>CwmsDataAccessConfigurator</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsDataAccessConfigurator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>703</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DatasourceTypeChangedSlot()</slot>
  <slot>OKClickedSlot()</slot>
 </slots>
</ui>
